ddtrace/tracer/util.go

```diff
@@ -27,6 +29,9 @@ func toFloat64(value interface{}) (f float64, ok bool) {
 	case int32:
 		return float64(i), true
 	case int64:
+		if i > max || i < min {
```

Comment:
My bad in describing the context. `max` and `min` are not inclusive. So please use `>=` and `<=`.